deno/payloads/v10/channel.ts

```diff
@@ -879,6 +883,20 @@ export enum MessageFlags {
 	IsVoiceMessage = 1 << 13,
 }
 
+/**
+ * https://discord.com/developers/docs/resources/channel#message-call-object-message-call-object-structure
+ */
+export interface APIMessageCall {
+	/**
+	 * Array of user object ids that participated in the call
+	 */
+	participants: Snowflake[];
+	/**
+	 * Time when call ended
```

Comment:
It would be nice to state what kind of `number` this is:

```suggestion
	 * ISO 8601 timestamp when call ended
```

Reply:
Good catch, I'll commit a fix in a moment